README.md

```diff
@@ -1,4 +1,4 @@
-## Welcome to awesome game of Tetris
+## Welcome to Github games
 
```

Comment:
Left reviews and line comments
Created a file and made a commit
Opened a pull request